internal/generator/vector/output/loki/ca-file.go

```diff
@@ -0,0 +1,18 @@
+package loki
+
+import (
+	"github.com/openshift/cluster-logging-operator/internal/generator/vector/output/security"
+)
+
+type CAFile security.CAFile
+
+func (ca CAFile) Name() string {
+	return "elasticsearchCAFileTemplate"
```

Comment:
@vimalk78: can you please change name to `lokiCAFileTemplate`

Reply:
changed